tests/scripts/webhook-config.yaml

```diff
@@ -9,12 +9,34 @@ webhooks:
       - apiGroups:   ["ceph.rook.io"]
         apiVersions: ["v1"]
         operations:  ["CREATE","UPDATE","DELETE"]
-        resources:   ["*"]
+        resources:   ["cephclusters"]
     clientConfig:
       service:
         name: ${SERVICE_NAME}
         namespace: ${NAMESPACE}
-        path: /validate
+        path: /validate-cephclusters
+      caBundle: ${CA_BUNDLE}
+    admissionReviewVersions: ["v1beta1"]
+    sideEffects: None
+    timeoutSeconds: 5
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: rook-cephblockpool-webhook
+  namespace: ${NAMESPACE}
+webhooks:
+  - name: ${SERVICE_NAME}.${NAMESPACE}.svc
+    rules:
+      - apiGroups:   ["ceph.rook.io"]
+        apiVersions: ["v1"]
+        operations:  ["CREATE","UPDATE","DELETE"]
+        resources:   ["cephblockpools"]
```

Comment:
Defining a separate ValidatingWebhookConfiguration for every CRD is really painful. Can you point to other examples of where this is done in other projects? I would really like to confirm if we can have one definition and watch all CRDs in the apiGroup `ceph.rook.io`

Reply:
I've updated the code. We can now make do with just one ValidatingWebhookConfiguration but we have to add a webhook in the array for each CRD and make sure that the path is written with the same template

Reply:
Looks good since it is in the same webhook config and we can use a single endpoint.